tests/positive/files/units.go

```diff
@@ -259,3 +260,38 @@ func TestMaskUnit() types.Test {
 		ConfigMinVersion: configMinVersion,
 	}
 }
+
+// RemoveEnablementSymLinksforUnit checks if Ignition
+// removes the enablement symlink for a given systemd
+// unit before disbling it.
+func RemoveEnablementSymLinksforUnit() types.Test {
```

Comment:
I'm not super familiar with blackbox tests, but I don't think this test is correct. I think you need to create the symlink in `in`, and then verify that it was removed in `out` (looks like there's `RemovedNodes` for that).

Reply:
This is still not quite right I think. `in` should have the enablement symlink too, not just the service file itself, and `out` should only verify that the symlink has been removed.

Reply:
> `in` should have the enablement symlink too, not just the service file itself, and out should only verify that the symlink has been removed.

I have added an enablement symlink in `in`, but it seems like the test is erroring out with this error: https://github.com/coreos/ignition/blob/main/tests/validator.go#L204

Reply:
Just wanted to confirm that this patch works fine if I use an Ignition config listed in https://github.com/coreos/fedora-coreos-tracker/issues/392